examples/vision/integrated_gradients.py

```diff
@@ -0,0 +1,474 @@
+"""
+Title: Integrated Gradients
+Author: [A_K_Nain](https://twitter.com/A_K_Nain)
+Date created: 2020/06/2
+Last modified: 2020/06/2
+Description: How to obtain integrated gradients for a classification model.
+"""
+
+"""
+## Integrated Gradients
+[Integrated Gradients](https://arxiv.org/abs/1701.07875) is a technique for
```

Comment:
is a model interpretability technique: you can use it to visualize the relationship between input features and model predictions.